grapher/mapCharts/MapChart.tsx

```diff
@@ -326,8 +326,7 @@ export class MapChart
         const { mapConfig, mapColumn, colorScale } = this
 
         return (d: PrimitiveType): string => {
-            if (isString(d)) return d
-            else if (mapConfig.tooltipUseCustomLabels) {
+                if (isString(d) || mapConfig.tooltipUseCustomLabels) {
```

Comment:
This will _always_ show the custom label (if there is one), no matter whether the checkbox is checked or not.

Instead, it would be better if you can do it in the following way:
``` js
if (mapConfig.tooltipUseCustomLabels) { /* ... */ }
else if (isString(d)) { /* ... */ }

return mapColumn?.formatValueLong(d) ?? ""
```